src/testing/testing.go

```diff
@@ -488,12 +505,15 @@ func (t *T) Run(name string, f func(t *T)) bool {
 	}
 
 	// Create a subtest.
+	ctx, cancelCtx := context.WithCancel(context.Background())
```

Comment:
It seems kinda weird that the parent context is not from the parent `testing.T`, but this seems to be how it is in the Go stdlib.

Reply:
Oh wait, there's specifically a comment on this in the Go stdlib:

> There's no reason to inherit this context from parent. The user's code can't observe the difference between the background context and the one from the parent test.